server/events/command/context.go

```diff
@@ -30,8 +30,9 @@ type Context struct {
 	Pull     models.PullRequest
 	Scope    tally.Scope
 	// User is the user that triggered this command.
-	User models.User
-	Log  logging.SimpleLogging
+	User   models.User
+	Log    logging.SimpleLogging
```

Comment:
can you remove this? Why have both? We're gonna need to do this for the whole codebase at some point anyways, might as well do it per file as we come across it.